scripts/globals/spells/healing_spell.lua

```diff
@@ -0,0 +1,190 @@
+-----------------------------------
+-- Healing Spell
+-- Used for spells that heal directly.
+-----------------------------------
+require("scripts/globals/spell_data")
+require("scripts/globals/jobpoints")
+require("scripts/globals/magicburst")
+require("scripts/globals/status")
+require("scripts/globals/utils")
+require("scripts/globals/msg")
+require("scripts/globals/magic")
+require("scripts/globals/settings")
+-----------------------------------
+xi = xi or {}
+xi.spells = xi.spells or {}
+xi.spells.healing = xi.spells.healing or {}
+
+xi.spells.healing.healingTable =
+{
+    [xi.magic.spell.CURE]           = { { power = 60,  maxCap = 20,  rate = 1,      constant = -10, minCap = 10  }, { power = 100, maxCap = 20,  rate = 2, constant = 5,    minCap = 10  }, { power = 999, maxCap = 30,  rate = 57,      constant = 29.125,   minCap = 10  } },
+    [xi.magic.spell.CURE_II]        = { { power = 110, maxCap = 75,  rate = 1,      constant = 20,  minCap = 60  }, { power = 170, maxCap = 75,  rate = 2, constant = 47.5, minCap = 60  }, { power = 999, maxCap = 90,  rate = 35.6666, constant = 87.62,    minCap = 60  } },
+    [xi.magic.spell.CURE_III]       = { { power = 180, maxCap = 160, rate = 1,      constant = 70,  minCap = 130 }, { power = 300, maxCap = 160, rate = 2, constant = 115,  minCap = 130 }, { power = 999, maxCap = 190, rate = 15.6666, constant = 180.43,   minCap = 130 } },
+    [xi.magic.spell.CURE_IV]        = { { power = 220, maxCap = 330, rate = 0.6666, constant = 165, minCap = 270 }, { power = 460, maxCap = 330, rate = 2, constant = 275,  minCap = 270 }, { power = 999, maxCap = 390, rate = 6.5,     constant = 354.6666, minCap = 270 } },
+    [xi.magic.spell.CURE_V]         = { { power = 320, maxCap = 570, rate = 0.6666, constant = 330, minCap = 450 }, { power = 560, maxCap = 570, rate = 1, constant = 410,  minCap = 450 }, { power = 999, maxCap = 690, rate = 2.8333,  constant = 591.2,    minCap = 450 } },
+    [xi.magic.spell.CURAGA]         = { { power = 110, maxCap = 75,  rate = 1,      constant = 20,  minCap = 60  }, { power = 170, maxCap = 75,  rate = 2, constant = 47.5, minCap = 60  }, { power = 999, maxCap = 90,  rate = 35.6666, constant = 87.62,    minCap = 60  } },
```

Comment:
For curagas does it make more sense to use the cure + 1 formula and then for blue magic use the values plus/minus the constants for each param? I feel like we're repeating ourselves a lot here

Reply:
This is exactly what we are doing, however this is following the same format as damage_spell.lua hence why it is done this way.